sale_outplacement/views/project_views.xml

```diff
@@ -8,29 +8,47 @@
       <field name="email_from" position="after">
         <field name="outplacement_id" />
       </field>
+      <field name="date_deadline" position="replace">
+        <field name="date_deadline" widget="date"/>
+      </field>
       <field name="date_deadline" position="before">
-        <field name="start_date"/>
+        <field name="start_date" widget="date"/>
       </field>
-      <field name="name" position="after">
+      <field name="project_id" position="before">
         <field name="instructions"/>
       </field>
+      <field name="project_id" position="attributes">
+        <attribute="invisible">1</attribute>

```

Comment:
Syntax error, <attribute="invisible">1</attribute> shall be <attribute name="invisible">1</attribute>

Same on several fields below